File: web3rpc/rpc-specs/code-samples/javascript/kaia/account/encodeAccountKey.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const accountKey = { keyType: 0, key: {} };

  provider.kaia
    .encodeAccountKey(accountKey, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/account/encodeAccountKey.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

accountKey = {
    "keyType": 0,
    "key": {}
}

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.encode_account_key(accountKey)

print(kaia_response)
